src/users/users.controller.ts

```diff
@@ -1,15 +1,15 @@
 import { Body, Controller, Post, UseGuards } from '@nestjs/common';
 import { UsersService } from './users.service';
-import { UserDto } from './user.dto';
-import { AuthGuard } from 'src/auth/auth.guard';
+import { UserDto, UserResponseDto } from './user.dto';
+import { AuthGuard } from 'src/guards/auth.guard';
 
 @UseGuards(AuthGuard)
-@Controller('users')
+@Controller('createUser')
```

Comment:
keep it as `users` 
you have a POST method that will be used to create the user
you can also extend later this user's controller to support more methods, such as GET, PUT, DELETE...